packages/vendure-plugin-campaign-tracker/src/entities/order-campaign.entity.ts

```
@@ -0,0 +1,20 @@
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { Campaign } from './campaign.entity';
+
+/**
+ * @description
+ * This entity represents the relation between an Order and a Campaign, including the connectedAt date
+ */
+@Entity()
+export class OrderCampaign extends VendureEntity {
+  constructor(input?: DeepPartial<OrderCampaign>) {
+    super(input);
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)